docs/src/languages/php/frameworks.md

```diff
@@ -38,7 +38,6 @@ Drupal FAQs, how-to guides and other tutorials right on [Platform.sh Community](
 ### Symfony templates
 
 * [Symfony 5](https://github.com/symfonycorp/platformsh-symfony-template/tree/5.4)
-* [Symfony 6](https://github.com/symfonycorp/platformsh-symfony-template/tree/6.0)
```

Comment:
:clipboard: The new version is 6.1, so we can link to that rather than deleting the line entirely: https://github.com/symfonycorp/platformsh-symfony-template/tree/6.1

Reply:
Oh sorry, I didn't look in the right place to see if there were other versions!